roles/migrationcontroller/defaults/main.yml

```diff
@@ -44,3 +44,14 @@ velero_restic_restore_helper_image: "{{ registry }}/{{ project }}/{{ velero_rest
 velero_restic_restore_helper_repo: "{{ lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_REPO') }}"
 velero_restic_restore_helper_version: "{{ snapshot_tag | default(lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_TAG')) }}"
 velero_state: absent
+
+# MCG integration
+noobaa: true
```

Comment:
We need to address what we want to support in non-OLM environments. The default here is noobaa: true and on openshift 3 there is nothing to install noobaa for the user. If they don't make a change I believe the operator is going to fail on install waiting for noobaa to come up for multiple tasks in this block:

https://github.com/fusor/mig-operator/pull/176/files#diff-c11bd2afe7682458145a58de7d973b7fR34